Dockerfile.ubi

```diff
@@ -17,26 +17,15 @@ USER root
 WORKDIR /work
 
 RUN microdnf install -y \
-        bzip2 \
-        curl-devel \
-        g++ \
         glibc-static \
         golang-bin \
-        libarchive-devel \
-        libmicrohttpd-devel \
-        libseccomp-static \
-        m4 \
         make \
-        sqlite-devel \
-        tar \
-        zlib-static
+        libseccomp-static
 
 ADD . /work
-RUN hack/install-libelf.sh
-RUN hack/install-libbpf.sh
 RUN mkdir -p build
 
-RUN make
+RUN WITHOUT_BPF=true make
```

Comment:
@jhrozek It may be clearer if the variable had a positive name, as to avoid double negation which can be confusing (i.e. `WITHOUT_BPF=false`).

On PR #680 I am proposing something similar for AppArmor. However, I called it `APPARMOR_ENABLED`. I am happy either approach as long as we keep it consistent. Do you have any strong preference between `XX_ENABLED` vs `WITH_XX`?

Reply:
Done, thank you, this is indeed better readable.